homeassistant/components/jewish_calendar/service.py

```diff
@@ -14,17 +15,20 @@
     ServiceResponse,
     SupportsResponse,
 )
-from homeassistant.helpers import config_validation as cv
+from homeassistant.helpers.selector import LanguageSelector, LanguageSelectorConfig
 
-from .const import DOMAIN
+from .const import ATTR_DATE, ATTR_NUSACH, DOMAIN, SERVICE_COUNT_OMER
 
+SUPPORTED_LANGUAGES = {"en": "english", "fr": "french", "he": "hebrew"}
```

Comment:
Assuming `Language` is an enum, you can just make the values here instances of the enum